libs/dh/metering-point/data-access-api/src/lib/dh-processes-data-access-api.store.ts

```diff
@@ -14,18 +14,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { HttpErrorResponse, HttpStatusCode } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { ComponentStore, tapResponse } from '@ngrx/component-store';
 import { filter, map, Observable, switchMap, tap } from 'rxjs';
-import {
-  MeteringPointHttp,
-  Process,
-} from '@energinet-datahub/dh/shared/domain';
-import { HttpErrorResponse, HttpStatusCode } from '@angular/common/http';
+import { MeteringPointHttp } from '@energinet-datahub/dh/shared/domain';
 import { ErrorState, LoadingState } from './states';
+import { DhProcess } from '../../../domain/src/model/dh-process';
```

Comment:
issue: Lint will flag this line as an error because we're not allowed to import cross-library. In this case the import should most likely be 

```suggestion
import { DhProcess } from '@energinet-datahub/dh/domain';
```